util/bytes2sigchain.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"bufio"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/nknorg/nkn/pb"
+)
+
+// Base64ToHex convert base64 string input to hex string output
+func Base64ToHex(in []byte) (out []byte, err error) { // input format should be ':"balabala..."'
+	const prefix, suffix = ":\"", "\""
+	start, end := len(prefix), len(in)-len(suffix) // b64Str len not contain :" prefix and " suffix
+
+	// Decode Base64 string to bin
+	bin := make([]byte, base64.StdEncoding.DecodedLen(end-start))
+	n, err := base64.StdEncoding.Decode(bin, in[start:end])
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "base64 decode %s met erro %v\n", in[start:end-1], err)
```

Comment:
should this be in[start:end]?

Reply:
My fault :joy: 
I had updated my commit.